Vagrantfile

```diff
@@ -0,0 +1,183 @@
+require 'rbconfig'
+
+Vagrant.require_version(">= 1.7.0")
```

Comment:
Yeah, I inherited this from the upstream "opsworks-vm" project. 1.7 would probably work but I've only tested with 1.8